src/web/app/pages-session/session-submission-page/saving-complete-modal/saving-complete-modal.component.html

```diff
@@ -38,15 +38,16 @@ <h5 class="modal-title">
     Note that you can change your responses and submit them again any time before the session closes.
   </p>
   <p *ngIf="!hasFailToSaveQuestions">
-    You are encouraged to download the proof of submission, which will also contain your latest responses.
+    <b>You are encouraged to download the proof of submission, which will also contain your latest responses.</b>
   </p>
 </div>
 <div class="modal-footer">
-  <button type="button" class="btn btn-info"
-          (click)="downloadProofOfSubmission()"
-          [disabled]="isAllQuestionSavingFailed">Download proof of submission</button>
-  <button type="button" class="btn modal-btn-ok"
+  <button type="button" class="btn modal-btn-ok btn-info"
           [ngClass]="
                 {'btn-danger': hasFailToSaveQuestions, 'btn-success': !hasFailToSaveQuestions}"
-          (click)="activeModal.close()">OK</button>
+          (click)="activeModal.close()">Don't download proof of submission</button>
+  <button type="button" class="btn btn-success"
+          (click)="downloadProofOfSubmission()"
+          [disabled]="isAllQuestionSavingFailed">Download proof of submission</button>
+
```

Comment:
A minor nit: there seems to be an extra blank line here, could you remove this to keep things neat and consistent?